packages/core/streams/index.test.ts

```diff
@@ -37,6 +37,31 @@ describe('AIStream', () => {
         )
       })
 
+      it('should support { mode: "raw" }', async () => {
+        const stream = OpenAIStream(
+          await fetch(server.api, {
+            headers: {
+              'x-mock-service': 'openai',
+              'x-mock-type': 'chat'
+            }
+          }),
+          { mode: 'raw' }
+        )
+
+        const response = new StreamingTextResponse(stream)
+        const client = createClient(response)
+
+        const rawChunks = await client.readAll()
+        const chunks = rawChunks.map(chunk => JSON.parse(chunk))
```

Comment:
In `raw` mode, the streams remain `ReadableStream<Uint8Array>` and the chunks come back as JSON strings. Here, they are parsed first simply so we don't stringify twice and have a bunch of escaped quotes.